src/javascripts/components/planesList/planesList.js

```diff
@@ -0,0 +1,22 @@
+import planesData from '../../helpers/data/planesData';
+import planesComponent from '../planes/planes';
+import utils from '../../helpers/utils';
+@import "../javascripts/components/planes/planes.scss";
```

Comment:
Ah my mistake. This’ll need to be in planes.js, then you can do `import ‘./planes.scss’`

Rule of thumb: import scss files into the JavaScript components the live “next” to. So planes.scss is imported by planes.js, planesList.scss by planesList.js, etc.

Reply:
Also you’ll note that you won’t use the @ symbol, that’s only if you’re importing in a scss file.